### PR TITLE
API-2199: Allow users to sign in with user1/password1 credentials

### PR DESCRIPTION
**Pull request by gokyo**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/testuser/services/TestUserServiceSpec.scala [hunk 1]
```diff
@@ -115,6 +116,12 @@ class TestUserServiceSpec extends UnitSpec with MockitoSugar {
       val org = await(underTest.authenticate(AuthenticationRequest(username, "P")))
       org shouldBe None
     }
+
+    // legacy sandbox user (user1/password1)
+    "Give the legacy sandbox user details if I authenticate with the legacy sandbox credentials" in new Setup {
+      val org = await(underTest.authenticate(sandboxAuthenticationRequest))
+      org shouldBe Some(sandboxUser)
```
Comment (tom59) [inline test/unit/uk/gov/hmrc/testuser/services/TestUserServiceSpec.scala]: a bit confusing to call it org.
can we can it individual?

Comment (tom59) [reply test/unit/uk/gov/hmrc/testuser/services/TestUserServiceSpec.scala]: or user

File: test/it/uk/gov/hmrc/TestUserSpec.scala [hunk 2]
```diff
@@ -81,15 +81,14 @@ with GivenWhenThen with BeforeAndAfterEach with BeforeAndAfterAll {
...
+      val prettyPayload = s"""
+                            |{
+                            |  "username": "$usr",
+                            |  "password" :"$pwd"
+                            |}
+                            |
+                           """
+      val payload = Json.parse(prettyPayload.stripMargin).toString
 
       Http(s"$serviceUrl/authenticate")
```
Comment (edinhodzic) [inline test/it/uk/gov/hmrc/TestUserSpec.scala:84]: For readability and general tidiness, here we could just go with:
```
Http(s"$serviceUrl/authenticate")
  .headers("Content-type" -> "application/json")
  .postData(s""" {"username": "$usr", "password" :"$pwd"} """).asString
```
In the process we remove both `payload` and `prettyPayload` along with the whitespace it currently creates and the unnecessary "string -> json -> string" conversions. A similar thing could be said for `expectedIndividualJson` further down in this class.

Comment (gokyo) [reply test/it/uk/gov/hmrc/TestUserSpec.scala:84]: I agree with you @edinhodzic in the case of a short json
In the case of `expectedIndividualJson` I prefer to keep it as it is

**Pull request merged**